core/src/main/java/org/apache/accumulo/core/clientImpl/CompactionStrategyConfigUtil.java

```diff
@@ -18,23 +18,17 @@
  */
 package org.apache.accumulo.core.clientImpl;
 
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
 import java.io.DataInput;
-import java.io.DataInputStream;
 import java.io.DataOutput;
-import java.io.DataOutputStream;
-import java.io.IOException;
-import java.util.HashMap;
 import java.util.Map;
-import java.util.Map.Entry;
 
+import org.apache.accumulo.core.client.admin.CompactionConfig;
 import org.apache.accumulo.core.client.admin.CompactionStrategyConfig;
 
+@SuppressWarnings("removal")
```

Comment:
This entire class is utility code a deprecated class.

Reply:
In that case, it too can be marked `@Deprecated` instead of merely suppressing the deprecation.

Reply:
I think other non-deprecated code calls this to deal with compaction strategies.  I was trying to put the code that does this in one place and in order to avoid suppression elsewhere.

Reply:
@ctubbsii does my answer address your concern?